scanomatic/server/analysis_effector.py

```diff
@@ -108,6 +110,10 @@ def total(self) -> int:
             return self._first_pass_results.total_number_of_images
         return -1
 
+    @property
+    def ready_to_start(self) -> bool:
```

Comment:
Du använder inte denna..?

Reply:
in the new test. wanted to have a separate test to show setup is ready

Reply:
ah, right. saw the test, but was wondering if it was yagni since you don't use it operationally.